src/data/products.ts

```diff
@@ -0,0 +1,18 @@
+export const mockProduct = {
+  id: 123,
+  name: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+  imageURL:
+    'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+  price: {
+    basicPrice: 29000,
+    discountRate: 0,
+    sellingPrice: 29000,
```

Comment:
큰 숫자를 사용할 때는 numeric seprator를 이용해 가독성을 높여보시면 좋겠네요
```suggestion
    basicPrice: 29_000,
    discountRate: 0,
    sellingPrice: 29_000,
```

Reply:
원 데이터를 그대로 사용했었는데 그런 방법도 있었네요..! 앞으로 큰 수를 사용할 일 있을 때마다 적용해보겠습니다!